notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -170,7 +170,7 @@
       "source": [
```

Comment:
<p>auto -&gt; automatic</p><p>configs -&gt; configurations</p><p> </p><p>Something appears wrong with the link:</p><p>[here](<a href="https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0" target="_blank">https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0</a>.31/google_cloud_pipeline_components.experimental.automl.tabular.html#google_cloud_pipeline_components.experimental.automl.tabular.FeatureTransformEngineOp).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Done. I double-checked the link & it is correct as it is intended to link directly to the specific component documentation.